README.md

```diff
@@ -0,0 +1,106 @@
+# 🌱  Fresh environment
+
+A **Fresh** environment is a fast and ready-to-use Docker container with
+various developer tools pre-installed. Including Node.js, and headless
+browsers. It aims to help to run npm packages on your machine,
+_without_ putting your personal data at risk!
+
+## Quick start
+
+Run the following from a terminal:
+
+```sh
+curl -s https://raw.githubusercontent.com/wikimedia/fresh/19.09.0/bin/fresh-node10 \
```

Comment:
I know this is a common way to install github stuff, especially lightweight code like shell scripts, but it might make sense to provide a checksum step here since, well, one of the issues fresh-node attempts to address is random, untrusted code :)

Reply:
It currently uses a Git tag, which is technically mutable indeed. Using a commit hash would make it more obvious that it isn't (easily) mutable. Could do a checksum as well. Do you think we should do both?

Can you recommend a minimal way to cross-platform do a checksum?

Reply:
I don't know if I'd have a great recommendation beyond the standard 1) publish file/repo hashes somewhere "trusted" 2) provide instructions for users to optionally validate the hashes in the README :/